### ISSUE TITLE
[Question] Domain shift result

### ISSUE THREAD

**Issue by moonsh** [juhongm999/hsnet#19]
COCO to Pascal - test

I tried to get domain shift results from your pre-trained model (Resnet50 COCO fold 0 ) but strangely got low accuracy.

Please correct me if I am wrong.

For example for the fold 0 in the domain shift test, 
First, I collect all train and validation pascal data and then exclude all classes that are in the coco fold 0.
Only Airplane, Boat, Chair, Dining table, Dog, Person remain for fold 0 as explained in the paper RePRI.

The entire above-remained fold 0 data were used for checking accuracy.
There are 7717 cases for the fold 0 test.

Should I sample 1000 cases from 7717 cases?

Thank you.

**Comment by juhongm999** [comment]
I shared the code for COCO->PASCAL evaluation.
You might want to refer to the data/coco2pas.py

**Comment by moonsh** [comment]
This helps me a lot. 
Thank you very much!
